python_modules/dagster/docs/intro_tutorial/dagstermill.rst

```diff
@@ -0,0 +1,278 @@
+============
+Dagstermill 
+============
+
+A wonderful feature of using Dagster is that you can productionize Jupyter notebooks and involve them in a (production) pipeline as units of computation. 
+
+There are a few stages of data scientists using notebooks in the wild. 
+
+1. Unstructured scratch work, cells are often run out of order.
+2. More refined prototyping, where cells are run sequentially. Usually the top cells contain parameters that are used in later cells. 
+3. Pieces of re-usable code are extracted from a notebook, turned into functions and put in a script (``.py`` file)
+
+Typically, only stage 3 would be involved in a production pipeline. However, with dagstermill, if you have a notebook in stage 2 (i.e. cells run sequentially to produce the desired output), with minimal effort you can register this notebook as a solid in the pipeline and use the notebook driven solid as a unit of computation that takes in inputs and produces outputs (that can be consumed by later stages of the pipeline).
+
+------------------------------------------
+An (Very Simple) Pipeline with a Notebook
+------------------------------------------
+
+Say you have a pipeline as shown below:
+
+.. image:: test_add_pipeline.png
+
+This is a very simple pipeline, where the solid ``return_one`` returns 1 and ``return_two`` returns 2. The notebook driven solid ``add_two_numbers`` simply adds two numbers and returns the result, as the name suggests.
+
+The notebook might have a cell that looks like the following:
+
+.. code-block:: python
+
+    a = 3
+    b = 5
+    result = 3 + 5
+
+Currently your notebook simply adds together the numbers ``3`` and ``5``, but in a more generic sense, your notebook is effectively a function that takes in inputs ``a`` and ``b`` and products output ``result``. To use the language of the dagster abstraction, it is a solid with inputs ``a``, ``b`` of dagster-type ``Int`` and with an ouput ``result`` of type ``Int``. 
+
+To register this notebook as a dagster solid, we use the following lines of code.
+
+.. code-block:: python
```

Comment:
So we the other parts of the tutorial have been doing which I think you should emulate here is actually include this code as a literal from another file and have it under test. This will also get you formatting for free (we do 79 character word wrap in the tutorials folder).